client/src/components/Categories/CategoriesOverview/category.css

```diff
@@ -6,7 +6,10 @@
     justify-content: space-around;
     flex-wrap: wrap;
     padding-bottom: 100px;
-   
+}
```

Comment:
you should not change links `a` for everything - just change it for category (make it under some class `.cateogry a` or something)

Reply:
also update for merge conflicts

Reply:
I do not understand what do you mean!!